docs/project-service-management-guide.md

```diff
@@ -156,106 +152,101 @@ deploy all manifests in this repository to the cluster in a set interval.
-Definition Repository. Additionally, each application inside the repository
-should also have an associated Azure DevOps multi-stage pipeline that both
-builds and deploys the latest Docker image to Azure Container Registry and
+(monorepository) applications depending on your development methodology.
+Typically, each repository shold be configured with a "hld-lifecycle" Azure
+DevOps pipeline that will add all managed applications inside the repository to
+the High Level Definition Repository. Additionally, each application inside the
+repository should also have an associated Azure DevOps multi-stage pipeline that
+both builds and deploys the latest Docker image to Azure Container Registry and
 updates the associated configuation in the HLD repository with the latest image
 tag.
 
 -TBD Section on packages directory and manging monorepositories vs single
 application repositories
 
-##### Initializing an Application Repository
+#### Initializing an Application Repository
 
 - [Create a repository in the given AzDO project.](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-new-repo?view=azure-devops#create-a-repo-using-the-web-portal)
 - [Clone the repository.](https://docs.microsoft.com/en-us/azure/devops/repos/git/create-new-repo?view=azure-devops#clone-the-repo-to-your-computer)
 - Initialize the project via `spk`
   ```
-  spk project init
-  git add -A
-  git commit -m "Initializing application repository."
-  git push -u origin --all
+  $ spk project init
+  $ git add -A
+  $ git commit -m "Initializing application repository."
+  $ git push -u origin --all
   ```
-- Create Variable Group via `spk` (optional flag parameters can be used if `spk`
-  was not intialized)
+- Create a variable group via `spk`. If you are using a repo per service source
+  control strategy, you only need to do this once.
   ```
-  VARIABLE_GROUP_NAME=<my-vg-name>
-  spk project create-variable-group $VARIABLE_GROUP_NAME -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
-  git add -A
-  git commit -m "Adding Project Variable Group."
-  git push -u origin --all
+  $ export VARIABLE_GROUP_NAME=<my-vg-name>
+  $ spk project create-variable-group $VARIABLE_GROUP_NAME -r $ACR_NAME -u $SP_APP_ID -t $SP_TENANT -p $SP_PASS
+  $ git add -A
+  $ git commit -m "Adding Project Variable Group."
+  $ git push -u origin --all
   ```
   where `ACR_NAME` is the name of the Azure Container Registry where the Docker
   Images will be served from and `SP_APP_ID`, `SP_PASS`, and, `SP_TENANT` are an
   associated Service Principal's ID, Password, and Tenant, that have Read and
   Write access to the ACR.
 - Deploy the lifecycle pipeline (optional flag parameters can be used if `spk`
   was not intialized)
+
   ```
-  spk project install-lifecycle-pipeline
+  $ spk project install-lifecycle-pipeline --org-name $ORG_NAME --devops-project $DEVOPS_PROJECT --repo-url $SERVICE_REPO_URL --repo-name $SERVICE_NAME --pipeline-name $SERVICE_NAME-lifecycle-pipeline
```

Comment:
`--pipeline-name $SERVICE_NAME-lifecycle-pipeline`

isn't strictly necessary, the default behavior would be <projectreponame>-lifecycle-pipeline .

Reply:
Ok I will drop it then.